Formula/c/cgal.rb

```diff
@@ -38,10 +37,12 @@ def install
       FindLASLIB.cmake
       FindMKL.cmake
       FindOSQP.cmake
-      FindOpenMesh.cmake
       FindSuiteSparse.cmake
     ]
     inreplace inreplace_files.map { |file| lib/"cmake/CGAL"/file }, "/usr/local", HOMEBREW_PREFIX
+
+    # These cause different bottles to be built between macOS and Linux for some reason.
+    %w[README.md readme.md].each { |file| (buildpath/file).unlink if (buildpath/file).exist? }
```

Comment:
I am sorry for that. Our test process missed that a few files were added to CGAL-6.0 root directory, and in particular a file `readme.md`. See https://github.com/CGAL/cgal/commit/0444375c58736ffeddfb8df536708785425683c8.

It will be fixed in CGAL-6.0.1, that I plan to publish soon.

Reply:
No problem; thanks for the update!